### PR TITLE
Fix: Add app_id to context Provider input

### PR DESCRIPTION
**Pull request by Meyanis95**
## Motivation

This PR enables React lib users to initialize their appId in the context Provider.

## Change Summary

- Add appId to Provider context
- Remove the v1 components
- Fix some UI bugs

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [x] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [x] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [x] PR includes documentation if necessary.

### CODE REVIEW

File: packages/anon-aadhaar-react/src/components/ProveButton.tsx [hunk 1]
```diff
@@ -1,66 +1,74 @@
...
-    },
-    app_id: {
-      argumentType: ArgumentTypeName.BigInt,
-      userProvided: false,
-      value: APP_ID,
-      description: '',
-    },
+  const startProving = async () => {
+    try {
+      if (appId === null) throw new Error('Missing application Id!')
```
Comment (vuvoth) [inline packages/anon-aadhaar-react/src/components/ProveButton.tsx:27]: If appId is null then client will throw error on console log?

File: packages/anon-aadhaar-react/src/provider/AnonAadhaarProvider.tsx [hunk 2]
```diff
@@ -22,16 +22,24 @@ import { SerializedPCD } from '@pcd/pcd-types'
...
+  const [pcd, setPcd] = useState<AnonAadhaarPCD | null>(null)
+  const [appId, setAppId] = useState<string | null>(null)
   const [state, setState] = useState<AnonAadhaarState>({
     status: 'logged-out',
   })
   useEffect(() => {
     readFromLocalStorage().then(setAndWriteState)
-  }, [])
+    setAppId(props._appId)
+    console.log(props._appId)
```
Comment (vuvoth) [inline packages/anon-aadhaar-react/src/provider/AnonAadhaarProvider.tsx:41]: I think we should have Loglevel for debug :D

File: packages/anon-aadhaar-react/src/components/ProveButton.tsx (hunk 1 above)
Comment (Meyanis95) [reply packages/anon-aadhaar-react/src/components/ProveButton.tsx:27]: Yes, and also in the error component displayed to the user.

File: packages/anon-aadhaar-react/src/provider/AnonAadhaarProvider.tsx (hunk 2 above)
Comment (Meyanis95) [reply packages/anon-aadhaar-react/src/provider/AnonAadhaarProvider.tsx:41]: Good catch, I missed this one!

**Review by vuvoth — APPROVED** [review]
LGTM!

**Pull request merged**